mqtt-clock.c

```diff
@@ -185,24 +179,34 @@ struct mosqagent_result* clock_idle(struct mosqagent *agent)
 }
 
 int main(int argc, char *argv[]) {
+    // set to non-time values
+    struct clock_state state = {
+        .current_second = 0xff,
+        .current_minute = 0xff,
+    };
+
+    struct mosqagent* agent;
+    agent = mosqagent_init_agent(&state);
+
```

Comment:
Check return value of `mosqagent_init_agent()`, or `mqtta_load_configuration()` in the next line may segfault and the following `printf()` as well.